online_store/gt_sports.html

```diff
@@ -6,94 +6,114 @@
     <title>goodTeens sports online store</title>
   </head>
   <body>
-<h2 class="column" id="h2title">goodTeens - Sports</h2>
+<h2 class="column" id="h2title">goodTeens - Sports</h2> <p>
 <h2 class="column" id="h2title2">Online Store</h2>
-<br><br><br><br><br>
-<p></p>
+<br><br>
+<p class="a"></p>
 <section class="card" id="product1"> 
-  <h3 class="productName1"></h3>
 </section>
+<p class="b"></p>
 <section class="card" id="product2"> 
-  <h3 class="productName2"></h3>
 </section>
+<p class="b"></p>
 <section class="card" id="product3"> 
-  <h3 class="productName3"></h3>
 </section>
+<p class="b"></p>
 <section class="card" id="product4"> 
-  <h3 class="productName4"></h3>
+</section>
+</section>
+<p class="b"></p>
+<section class="card" id="product5"> 
 </section>
 
 <script>
 const sportProducts = [
   {
-    imgSrc: "/skateboard.jpg",
+    imgSrc: "skateboard.jpg",
     name: "Skateboard",
     brand: "Hudora",
     level: "beginner",
     price: 50,
```

Comment:
You need to have at least one nested object. As an idea you can make price an object:
```javascript
price: {
    amount: 50,
    currency: "EUR"
}
```